keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -135,46 +149,47 @@ Scenario: [...]
 - **`${k8s_release}`**: the version of kubernetes `<major>.<minor>`
   (e.g. `1.12`, `1.13`, `1.14`, ...)
 - **`${channel}`**: can be `stable`, `dev`, `nightly`
-    - `stable`: all official releases for `${k8s_release}`
-      (e.g.: `1.13.0`, `1.13.1`, `1.13.2`, ...)
-    - `dev`: all development releases for all minor releases in this `${k8s_release}`, including `alpha`s, `beta`s and `rc`s
-      (e.g.: `1.13.0-rc.2`, `1.13.2-beta.0`, `1.13.1-alpha.3`, ...)
-    - `nightly`: any package cut automatically on a daily basis
-
+  - `stable`: all official releases for `${k8s_release}`
+    (e.g.: `1.13.0`, `1.13.1`, `1.13.2`, ...)
+  - `dev`: all development releases for all minor releases in this `${k8s_release}`, including `alpha`s, `beta`s and `rc`s
+    (e.g.: `1.13.0-rc.2`, `1.13.2-beta.0`, `1.13.1-alpha.3`, ...)
+  - `nightly`: any package cut automatically on a daily basis (optionally)
 
-This means, that end-users can configure their systems’ package managers to use those different `${channel}`s of a kubernetes `${k8s_release}` for their `${dist}`.
+This means, that End Users can configure their systems’ package managers to use
+those different `${channel}`s of a kubernetes `${k8s_release}` for their
+corresponding package manager.
 
 A configuration for the package managers might look something like:
 
 - deb:
-    ```
-    # deb http://apt.kubernetes.io/${dist} ${k8s_release} ${channel}
-    deb http://apt.kubernetes.io/debian-jessie 1.13 nightly
-    ```
+  ```
+  # deb http://apt.kubernetes.io ${k8s_release} ${channel}
+  deb http://apt.kubernetes.io/debian-jessie 1.13 nightly
+  ```
 - rpm/yum:
-    ```
-    [kubernetes]
-    name=Kubernetes
-    # baseurl=http://yum.kubernetes.io/${dist}/${k8s_release}/${channel}
-    baseurl=http://yum.kubernetes.io/fedora-27/1.13/nightly
-    enabled=1
-    gpgcheck=1
-    repo_gpgcheck=1
-    gpgkey=file:///etc/pki/rpm-gpg/kubernetes.gpg.pub
-    ```
+  ```
+  [kubernetes]
+  name=Kubernetes
+  # baseurl=http://yum.kubernetes.io/${k8s_release}/${channel}
+  baseurl=http://yum.kubernetes.io/fedora-27/1.13/nightly
+  enabled=1
+  gpgcheck=1
+  repo_gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/kubernetes.gpg.pub
+  ```
 
 Different architectures will be published into the same repos, it is up to the package managers to pull and install the correct package for the target platform.
 
+Ideally and optionally, publishing/promoting a package means to commit a change
+to a configuration file which triggers a "package promotion tool", which:
 
-Ideally, publishing/promoting a package means to commit a change to a configuration file which triggers a "package promotion tool".
-That tool ...
-- manages which packages need to go into which `${channel}` for which `${dist}` of which `${k8s_release}`
+- manages which packages need to go into which `${channel}` for which package manager of which `${k8s_release}`
 - guard that by the packages checksum
```

Comment:
I'd like to leave that out for now since it moves the KEP more into the SLSA direction (where we already have https://github.com/kubernetes/enhancements/issues/3027 for).